R/rcpp.R

```diff
@@ -25,6 +26,41 @@ use_rcpp <- function(name = NULL) {
   invisible()
 }
 
+#' @rdname use_rcpp
+#' @export
+use_rcpp_armadillo <- function(name = NULL) {
+  use_rcpp(name)
+
+  use_dependency("RcppArmadillo", "LinkingTo")
+
+  makevars_path <- proj_path("src/Makevars")
+  makevars_win_path <- proj_path("src/Makevars.win")
+
+  makevars_settings <- c(
+    "CXX_STD = CXX11",
+    "PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)",
+    "PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS) $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)"
+  )
+
+  create_makevars(makevars_path, "RcppArmadillo", makevars_settings)
+  create_makevars(makevars_win_path, "RcppArmadillo", makevars_settings)
+
+  invisible()
+}
+
+#' @rdname use_rcpp
+#' @export
+use_rcpp_eigen <- function(name = NULL) {
+  use_rcpp(name)
+
+  use_dependency("RcppEigen", "LinkingTo")
+  use_dependency("RcppEigen", "Imports")
+
+  roxygen_ns_append("@import RcppEigen") && roxygen_update()
```

Comment:
Can you briefly explain to me why RcppEigen needs to be imported but RcppArmadillo does not?

Reply:
Short Version: RcppEigen doesn't need Imports and this has been removed.

Longer version: This was an artifact of how `RcppEigen.package.skeleton()` created a new package. As a result, I've issued a patch for the RcppEigen project that addresses it and we've incorporated the change here.